### DISCUSSION TITLE
Should PascalABC.NET go native?

### DISCUSSION THREAD

**Discussion by ghost** [pascalabcnet/pascalabcnet#2951] [General]
Recently, I discovered and spent my time on Vala. It's a C#-like language but compiles to native code. It offers Collections-like containers by leveraging GLib and it has seamless integration with GNOME. It's truly cross-platform. I'm happy with it.

**Comment by ibond84** [comment]
PABC.NET has the experimental feature {$platformtarget native} (for console apps only). 
.NET 7.0 is required.

**Comment by ibond84** [comment]
>AOT support in .NET 8 is more comprehensive than in .NET 7. However, there are still some limitations.

We wait for .NET 8 :)
